### ISSUE TITLE
Not Working on Handheld mode?

### ISSUE THREAD

**Issue by amarantsnipersnake** [Sopsy/joycon-turnoff#1]
after i use this its working and i see the joycon is shut down. but when i attach it to nintendo switch using a handheld mode. and when i remove the joycon again. the joycon is searching again when i press any button. are this app use for joycon wireless use only?

**Comment by victorkte** [comment]
this tool is meant to use when you are storing your joycons and don't intend to use them again
if you attatch it to a switch it will get off of the deep sleep state and work as it did before, if you want to completely shut it down again you'll have to pair it to your computer and use this tool again

**Comment by Sopsy** [comment]
Yup, as @victorkte explained.

This is not preventable, as the controller is switched off of "Shipping Low Power Mode" as soon as it's reconnected to a Switch and this behavior can't be changed. This happens both when pairing wirelessly or when attaching to a Switch physically.

I created this tool to be able to preserve batteries on controllers I don't plan on using for years. And I'm happy to report that almost all of the controllers I turned off with this tool three years ago still have battery left, they still turn on when sync button is held down. Not having done this it is very possible the batteries would have been completely dead by now and unable to charge anymore.
